godot-macros/src/docs.rs

```diff
@@ -26,14 +26,14 @@ pub fn make_definition_docs(
             .filter_map(member)
             .collect::<String>();
         Some(quote! {
-            docs: ::godot::docs::StructDocs {
+            ::godot::docs::StructDocs {
                 base: #base_escaped,
                 description: #desc_escaped,
                 members: #members,
-            }.into()
+            }
         })
     })()
-    .unwrap_or(quote! { docs: None })
+    .unwrap_or(TokenStream::new())
```

Comment:
Damn, we still have this `(|| { ... })()` pattern :grin: 

I need to make a note to clean this up at some point...

Reply:
this actually doesnt work properly, i didnt check with `register-docs` enabled at first so i missed that this was wrong

Reply:
i replaced this occurence with a let-else while i was at it